layouts/advocate.vue

```diff
@@ -0,0 +1,423 @@
+<template>
```

Comment:
I have tried integrating the code in `~/pages/advocates/index.vue` rather than in `~/layouts/advocate.vue`. But what I've understood is that the code in `~/pages/advocates/index.vue` works like a recursion where the code is called over and over again corresponding to each card / markdown file and no more than one section could be added. 

My idea was to keep everything except `Meet the advocates` to be the main element.

Please let me know if you have a different idea.

Reply:
Layouts are intended to hold the shared components of several pages (https://nuxtjs.org/guide/views#layouts) while the pages contain the actual content. That recursion you mention does not prevent you from adding more content. We can combine pure HTML with Vue components and directives. Don't worry too much about the dynamic part and try to move the contents of this file to `~/pages/advocates/index.vue`.